src/commcare_cloud/ansible/roles/deploy_hq/tasks/run_webpack.yml

```diff
@@ -1,10 +1,20 @@
 - name: Check for webpack configuration
+  # We can eventually remove this when all of HQ is on Webpack
   stat:
-    path: "{{ code_source }}/webpack.config.js"
-  register: config_file 
+    path: "{{ code_source }}/webpack"
+  register: webpack_config_directory
 
-- name: Run webpack
+- name: Generate Webpack Settings
+  # generates webpack/settings.json so that webpack has the same path
+  # to STATIC_ROOT as django
   command:
-    cmd: yarn run build:webpack
+    cmd: ./manage.py generate_webpack_settings
```

Comment:
So, if we were to want to do this process manually, we'd need to call `./manage.py generate_webpack_settings` followed by `yarn build`? I realize you've created `generate_webpack_settings` to split off this work, but is this not something that can be done as part of `yarn build`?

Reply:
i wanted to separate a management command from django with the `yarn build` process.